tests/__init__.py

```diff
@@ -35,3 +42,41 @@ def unreadable_pdf(tmp_path: Path) -> str:
     file_path = tmp_path / "document.pdf"
     file_path.touch(mode=0o000)
     return str(file_path)
+
+
+def get_qt_app() -> Application:
+    if Application.instance() is None:
+        return Application()
+    else:
+        return Application.instance()
```

Comment:
Could there be issues with reusing the same Application instance? I'm afraid some saved state could be interfere.

Reply:
I don't know either, that remains to be seen. I have the same reservation that you have as well, but I didn't find a better way to tackle this. Note that instantiating a new `Application` object will make the program abort.

Reply:
> whereas forcing the Application to exit does not seem to much.

Which method were you using to exit the application? There may be a way to deal with this. I can look it up.

Reply:
I think the `Application` has an `exit()` method, but calling it was not sufficient.

Reply:
Oh, wait a sec! I just realized that I didn't properly copy-paste the upstream issue I was talking about. I've edited my original comment, here it is: https://bugreports.qt.io/browse/PYSIDE-1190

Reply:
OK. I'll close this as there doesn't seem to be a solution.